src/main/solidity/com/ampnet/auditornode/contract/ApxCoordinator.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./shared/Structs.sol";
+
+interface ApxCoordinator {
+
+    // This dummy function this needed for Web3j to correctly generate structure class. This this because Web3j does not
```

Comment:
This this!!

Reply:
Good catch 😄

Reply:
Fun fact about this: IntelliJ auto-completes `is` into `this` for Solidity comments. 🤦  I guess that the Solidity plugin for IntelliJ is not too good...